apps/condo/domains/notification/constants/constants.js

```diff
@@ -401,6 +402,24 @@ const PUSH_FAKE_TOKEN_FAIL = 'PUSH_FAKE_TOKEN_FAIL'
 const FIREBASE_CONFIG_ENV = 'FIREBASE_CONFIG_JSON'
 const FIREBASE_CONFIG_TEST_PUSHTOKEN_ENV = 'FIREBASE_PUSH_TOKEN_TEST'
 
+const HCM_CONFIG_ENV = 'HCM_CONFIG_JSON'
+const HCM_CONFIG_TEST_PUSHTOKEN_ENV_RESIDENT = 'HCM_PUSH_TOKEN_TEST_RESIDENT'
+const HCM_CONFIG_TEST_PUSHTOKEN_ENV_MASTER = 'HCM_PUSH_TOKEN_TEST_MASTER'
+
+const APP_MASTER_KEY = 'master'
+const APP_RESIDENT_KEY = 'resident'
+const APP_RESIDENT_ID_ANDROID = 'ai.doma.client'
+const APP_RESIDENT_ID_IOS = 'ai.doma.clients'
```

Comment:
Values and keys are unobvious and don't self-explanatory

Reply:
We have several mobile apps, for resident and master on ios and android. Apps have fixed ids, that are described within that constants.